README.md

```diff
@@ -23,6 +23,8 @@ Now you can run the script by typing (You will need netcat installed on the comp
 
     ./init_and_wait_for_db.sh
 
+**You will need to modify config.example.py with the correct database parameters.**
```

Comment:
After further investigation, it does need to be modified if you change the root password or anything in the faf-db. Otherwise, the script will error out when being ran and display: 

Ignoring lupa import error: No module named 'lupa'
Traceback (most recent call last):
  File "run.py", line 21, in <module>
    api_init()
  File "/code/api/__init__.py", line 95, in api_init
    faf.db.init_db(app.config)
  File "/code/db/faf/db/__init__.py", line 13, in init_db
    connection = pymysql.connect(**config['DATABASE'])
  File "/usr/local/lib/python3.5/site-packages/pymysql/__init__.py", line 88, in Connect
    return Connection(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/pymysql/connections.py", line 679, in __init__
    self.connect()
  File "/usr/local/lib/python3.5/site-packages/pymysql/connections.py", line 891, in connect
    self._request_authentication()
  File "/usr/local/lib/python3.5/site-packages/pymysql/connections.py", line 1097, in _request_authentication
    auth_packet = self._read_packet()
  File "/usr/local/lib/python3.5/site-packages/pymysql/connections.py", line 966, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python3.5/site-packages/pymysql/connections.py", line 394, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.5/site-packages/pymysql/err.py", line 120, in raise_mysql_exception
    _check_mysql_exception(errinfo)
  File "/usr/local/lib/python3.5/site-packages/pymysql/err.py", line 112, in _check_mysql_exception
    raise errorclass(errno, errorvalue)
pymysql.err.OperationalError: (1045, "Access denied for user 'root'@'172.17.0.3' (using password: NO)")